en/guides/students/exercises/index.md

```diff
@@ -1,174 +1,169 @@
 ---
-title: "[nl] Oefeningen oplossen"
-description: "Tutorial: oefeningen oplossen"
+title: "Solving Exercises"
+description: "Tutorial: solving exercises"
 order: 4
 ---
 
-::: warning Sorry
-For now, this page is only available in Dutch. Sorry!
-:::
-
-# Oefeningen oplossen
-> Alle informatie zodat je als student kan navigeren naar oefeningen of oplossingen, oplossingen kan indienen en feedback kan interpreteren.
-
-**Inhoudsopgave**
-[[toc]]
+# Solving Exercises
+All the information you need to navigate to exercises or solutions, submit solutions, and interpret feedback as a student.
```

Comment:
```suggestion
> All the information you need to navigate to exercises or solutions, submit solutions, and interpret feedback as a student.
```

This adds the quote styling so it is more clear that it is a summary of what is found on the page

Reply:
This has been removed in all dutch versions, along with the table of contents in some previous sweep when we the style of the website was changed.
For some reason this change wasn't applied in the English duplicates of the Dutch files.


For consistency I wouldn't use this `>` style here as we also no longer use it on other pages

Reply:
But then this introductory sentence must also be removed, right?

Reply:
it isn't in the Dutch guides: https://docs.dodona.be/nl/guides/students/exercises/

And I do think the sentence has some value